lobsterpy/cohp/analyze.py

```diff
@@ -83,11 +87,16 @@ def __init__(
             are_cobis : bool indicating if file contains COBI/ICOBI data
             are_coops : bool indicating if file contains COOP/ICOOP data
             noise_cutoff : float that sets the lower limit of icohps or icoops or icobis considered
-            whichbonds: selects which kind of bonds are analyzed. "cation-anion" is the default
+            orbital_int_cutoff : float that sets the minimum percentage for the orbital resolved analysis.

```

Comment:
```suggestion
            orbital_cutoff : float that sets the minimum percentage for the orbital resolved analysis.

```

int suggests integer and this is confusing, I think.